akka-stream/src/main/scala/akka/stream/javadsl/Flow.scala

```diff
@@ -365,6 +365,11 @@ final class Flow[-In, +Out, +Mat](delegate: scaladsl.Flow[In, Out, Mat]) extends
     new Flow(delegate.grouped(n).map(_.asJava)) // TODO optimize to one step
 
   /**
+   * TODO: description
```

Comment:
when you document this please mention here `take` (which does not fail on the limit) and at the `take` docs mention this as the failing counterpart.